spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -371,18 +365,13 @@ private SparkContext createSparkContext_2() {
   }
 
   public SparkContext createSparkContext_1() {
-    logger.info("------ Create new SparkContext {} -------", getProperty("master"));
-
-    String execUri = System.getenv("SPARK_EXECUTOR_URI");
-    String[] jars = null;
-
-    if (Utils.isScala2_10()) {
```

Comment:
Do you mean enviroment variable `ADD_JARS` ? It is deprecated from spark 1.3 (SPARK-3340) and no spark document mention this variable, so I think it is safe to ignore it in zeppelin to keep the code clean (Actually I think we should drop support for old version of spark in some piont of future, otherwise the SparkInterpreter is hard to maintain ). If this is a concern, I can revert this piece of code.